gslib/utils/user_agent_helper.py

```diff
@@ -33,10 +33,10 @@ def GetUserAgent(args, metrics_off=True):
   user_agent = ' gsutil/%s' % gslib.VERSION
   user_agent += ' (%s)' % sys.platform
   user_agent += ' analytics/%s ' % ('disabled' if metrics_off else 'enabled')
-  user_agent += ' interactive/%s' % system_util.IsRunningInteractively()
+  # user_agent += ' interactive/%s' % system_util.IsRunningInteractively()
 
   if len(args) > 0:
-    user_agent += ' command/%s' % args[0]
+    """user_agent += ' command/%s' % args[0]
```

Comment:
Do you think it would be better to simply remove the commented code to keep it clean. We can always revert this PR at a later point to include the deleted code, or simply look at the diff of this PR.